tests/core/modules/conftest.py

```diff
@@ -0,0 +1,48 @@
+from itertools import product
+from random import choice
+import pytest
+import numpy as np
+from pymatgen.core.periodic_table import Element
+from pymatgen.core import Structure
+
+from fairchem.core.datasets import LMDBDatabase, AseDBDataset
+
+
+@pytest.fixture(scope="session")
+def dummy_element_refs():
+    # create some dummy elemental energies from ionic radii (ignore deuterium and tritium included in pmg)
+    return np.concatenate(
+        [[0], [e.average_ionic_radius for e in Element if e.name not in ("D", "T")]]
+    )
+
+
+@pytest.fixture(scope="session")
+def max_num_elements(dummy_element_refs):
+    return len(dummy_element_refs) - 1
+
+
+@pytest.fixture(scope="session")
+def dataset(tmpdir_factory, dummy_element_refs):
```

Comment:
nit; maybe this name should be more specific, fake_binary_dataset ? just to distinguish it from potentially other dataset test fixtures?

Reply:
Done!